src/userAttrCache.cpp

```diff
@@ -439,8 +428,14 @@ void UserAttrCache::fetchStandardAttr(UserAttrPair key, std::shared_ptr<UserAttr
     .then([wptr, this, key, item](ReqResult result)
     {
         wptr.throwIfDeleted();
-        item->data.reset(gUserAttrDescs[key.attrType].getData(*result));
-        item->resolve(key);
+
+        UserAttrDescMap::iterator it = gUserAttrDescsMap.find(key.attrType);
+        if (it != gUserAttrDescsMap.end())
+        {
+            auto& desc = it->second;
+            item->data.reset(desc.getData(*result));
+            item->resolve(key);
+        }
```

Comment:
Is it possible to fetch an attribute that it's not at the `gUserAttrDescsMap`?? I think it shouldn't. Let's discuss about it.

Reply:
I agree with you, we shouldn't fetch an attribute out of gUserAttrDescsMap

Reply:
Then I'll simplify it :)